src/backend/tools.ts

```diff
@@ -1,10 +1,10 @@
 import { WineInstallation } from 'common/types'
-import * as axios from 'axios'
+import Axios from 'axios'
```

Comment:
```suggestion
import axios from 'axios'
```
(we've imported it as "axios" just about everywhere else, but of course this is nitpicking)

Reply:
yes, never really understood why so we always need to use:
`axios.default.get` instead of only `Axios.get`

Reply:
Oh I'm aware, I was just commenting on the name (we've used lowercase "axios" everywhere else, but capitalized "Axios" here)

Reply:
very nitpicking really, fixed now :P